From e20f60934d5393e670537a8efc372b1daac69343 Mon Sep 17 00:00:00 2001
From: Samuel Ortiz <sameo@linux.intel.com>
Date: Fri, 24 Oct 2014 02:28:26 +0200
Subject: [PATCH 2/2] net: wireless: bcmdhd: Add 43340 ACPI IDs

The Vespa8 DSDT will give us either BCM43341 or BCM4334.

Change-Id: Icf33b7487b27576b212a30778beae633f2b072bd
Signed-off-by: Samuel Ortiz <sameo@linux.intel.com>
---
 drivers/net/wireless/bcmdhd/dhd_linux_platdev.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/net/wireless/bcmdhd/dhd_linux_platdev.c b/drivers/net/wireless/bcmdhd/dhd_linux_platdev.c
index b776838..25b5128 100644
--- a/drivers/net/wireless/bcmdhd/dhd_linux_platdev.c
+++ b/drivers/net/wireless/bcmdhd/dhd_linux_platdev.c
@@ -338,6 +338,8 @@ static struct acpi_device_id bcm_acpi_id[] = {
 /* ACPI IDs here */
 	{ "BCM43241" },
 	{ "BCM4321" },
+	{ "BCM43341" },
+	{ "BCM4334" },
 	{ }
 };
 MODULE_DEVICE_TABLE(acpi, bcm_acpi_id);
-- 
1.9.1

